tests/server/mocks/response.js

```diff
@@ -2,10 +2,12 @@
 
 var sinon = require('sinon');
 
-module.exports = {
-	writeHead: sinon.stub(),
-	write: sinon.stub(),
-	getHeader: sinon.stub(),
-	setHeader: sinon.stub(),
-	end: sinon.stub()
+module.exports = function () {
```

Comment:
All other mocks export an object rather than a function, I would prefer to keep this consistency.

Reply:
Exporting an object as a mock introduces coupling between test runs as node modules are singletons, so not a great plan. Happy to not include this if the tests still pass, but would recommend either adding it to the next commit and refactoring the mocks asap.

Reply:
I don't believe node modules _always_ behave like singletons, due to the ah... subtleties, let's say, of node's module caching and some other factors 😉 but you are right in this case, and make a good point.